docs/HowTo/BreakingChanges.md

```diff
@@ -0,0 +1,142 @@
+# Breaking Changes
+
+Compatibility is a very important goal of each .NET release.  In order to achieve this, various techniques are employed. As shown
+earlier, there is additive functionality with each version, so previous versions will still work.  However, when a change must be 
+made to previous functionality (for performance, security, or a bug) this can cause compatibility problems in already written code. 
+A distinction to understand with .NET compatibility is runtime vs retargeting issues.
+
+When developing an app, the .NET Framework version that the app is targeting is [specified either by specifying a target Framework 
+in Visual Studio](https://msdn.microsoft.com/en-us/library/bb398202%28v=vs.140%29.aspx), directly 
+[specifying the target Framework](https://msdn.microsoft.com/en-us/library/hh264221.aspx) in a project file, or applying a 
+[TargetFrameworkAttribute](https://msdn.microsoft.com/en-us/library/system.runtime.versioning.targetframeworkattribute%28v=vs.110%29.aspx) 
+to the source code.
+
+An app that is built targeting a particular version of the .NET Framework (4.0 or above) will run on that version of the .NET 
+Framework or on any newer in-place update version. When running on a newer version than what was targeted, the .NET Framework will 
+use quirked behavior to mimic the older targeted version. The app will run on the newer version of the Framework, but act as if it's
+running on the older version. Many of the breaking changes between versions of the .NET Framework are mitigated through this 
+quirking model.
+
+Runtime issues are when a new runtime is placed on a machine and the same binaries are run. If a binary was compiled for .NET 4.0,

```

Comment:
Runtime issues are when... this part seems a bit confusing to me. Maybe re-word it to be more like the retargeting bit that comes next (i.e. Runtime issues are those that occur when a new runtime is placed on a machine - regardless of which runtime is targeted...)